mupdf: add page (#5168)
tldr-pages/tldr: pages/common/mupdf.md

# mupdf

> > MuPDF is a lightweight PDF viewer (also supports XPS and E-Book formats).

Reviewer: Perhaps this will read better?
```suggestion
> MuPDF is a lightweight PDF viewer, with full support for PDF, XPS and E-Book formats.
```

Author: I am not sure it has full support. Also, I haven't really met anyone that uses it as a XPS or E-Book reader. How about:
MuPDF is a lightweight PDF, XPS, and E-book viewer?

This is from the website of MuPDF

Reviewer: Sounds perfect&mdash;
```suggestion
> MuPDF is a lightweight PDF, XPS, and E-book viewer.
```

> More information: <https://www.mupdf.com>.

- Open a PDF on the first page:

`mupdf {{filename}}`

- Open a PDF on page 3:

`mupdf {{filename}} {{3}}`

- Open a password secured PDF:

`mupdf -p {{password}} {{filename}}`

- Open a PDF with an initial zoom level, specified as DPI, of 72:

`mupdf -r {{72}} {{filename}}`

- Open a PDF with inverted color:

`mupdf -I {{filename}}`

> - Open a PDF tinted red FF0000 (hexadecimal color syntax RRGGBB):

Reviewer: How about the standard web color format...
```suggestion
- Open a PDF tinted red #FF0000 (hexadecimal color syntax RRGGBB):
```

Author: As a non-web developer I don't come across this syntax often, in other environments we would write 0xFF0000 instead. But I feel your suggestion is valid, I will accept the commit suggestions :)

Reviewer: I agree. I mainly suggested this because plain FF0000 doesn't look quite good to me, and 0xFF0000 doesn't particularly strike me as a color :)

Beside, when dealing with creating/editing PDFs, one will probably encounter hexadecimal colors quite frequently.
